### ISSUE TITLE
Focused window jitters every ~3s

### ISSUE THREAD

**Issue by Acumane** [hyprwm/Hyprland#3858]
### Hyprland Version

Hyprland, built from branch HEAD at commit 9afdd61adeaf413afbe853dbc5fdf6853c8bde21  (props: bump ver to 0.32.3). Tag: v0.32.3

### Bug or Regression?

Bug

### Description

Every ~3s, the focused window will "jitter": it briefly resizes and then returns to its initial state. Border color change suggests it's quickly being unfocused as well. Any open popups/context menus disappear during the event

[ex.webm](https://github.com/hyprwm/Hyprland/assets/94324138/9dab3c51-a77a-4128-9666-9cf852c2d1b5)

As per @FlashOnFire, who also reproed:
> tried both amd and nvidia gpus
> Issue don't happen on v0.30.0
> v0.31.0 won't build for some reason
> v0.32.0 and above are buggy 
> Nothing useful in logs

### How to reproduce

In a version <`v0.32.0` (or <`v0.31.0`?): focus window. Observe

**Comment by vaxerski** [comment]
something is opening a window. I don't think there is a bug here.

Inspect the logs to see what window is being open.

**Comment by FlashOnFire** [comment]
> something is opening a window. I don't think there is a bug here.
> 
> Inspect the logs to see what window is being open.

We talked about it on discord and it happens that we are both using espanso.

With latest hyprland, the espanso service tries to start in a loop and crashes instantly
There's nothing relevant in the hyprland log on my side but here's the espanso log when trying to start it manually :
https://pastebin.com/mExju11Y

It may or may not be an espanso issue but it's weird that it works reliably on previous versions of hyprland

**Comment by Acumane** [comment]
> Inspect the logs to see what window is being open.

[My logs](https://pastebin.com/sXArf3mz) caught Espanso red-handed, but as Flash mentions there are older versions where this doesn't happen.

The issue also crops up again some time before `v0.30.0`—at least according to [this issue ticket](https://github.com/espanso/espanso/issues/1649).

**Comment by h1tarxeth** [comment]
yup its Espanso i tried killing it but it failed then i uninstalled it and it was good to go for time being just uninstall it `sudo pacman -R espanso-wayland ` .

**Comment by vaxerski** [comment]
closing as it's not a hl issue
